app/models/concerns/managed_by.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ManagedBy
+  extend ActiveSupport::Concern
+
+  def managed_by
+    value_of_annotation("managed")
+  end
+
+  def managed_by=(value)
+    annotate("managed", value)
+  end
```

Comment:
These look good as helpers to get/set the annotation we care about here. but trying to treat that as an actual attribute, seems to be unnecessary complicated. I hope it's clear what I tried to say.